packages/components/src/styles/index.css

```diff
@@ -0,0 +1,116 @@
+/**
+ * @synergy-design-system/components version 1.26.0
+ * SICK Global UX Foundation
+ * 
+ * This file is created automatically in the build process!
+ * All changes applied will get lost!
+ * To recreate it, please use "pnpm build"!
+ */
+
+/**
+ * ---------------------------------------------------------------------
+ * 🔒 AUTOGENERATED BY VENDORISM
+ * Removing this comment will prevent it from being managed by it.
+ * ---------------------------------------------------------------------
+ */
```

Comment:
The index.css file is automatically created in the same vain as in the styles package. Maybe it would be better to create this in the `dist` directory only instead as it is the "bundled" version of all css files in `src/files`.

@mariohamann: What do you think?

Reply:
This is now changed to create the `index.css` file for `dist` only. This is fine as it is a virtual file only and should be treated as such.